### PR TITLE
Changed <b> tag to <strong tag - JSON challenge

### LINKED ISSUE #9586: Convert JSON Data to HTML should use 'strong' and self-closing br
Challenge [Convert JSON Data to HTML](https://www.freecodecamp.com/challenges/convert-json-data-to-html#?solution=%0Afccss%0A%20%20%24(document).ready(function()%20%7B%0A%0A%20%20%20%20%24(%22%23getMessage%22).on(%22click%22%2C%20function()%20%7B%0A%E2%80%89%E2%80%89%20%20%20%20%24.getJSON(%22%2Fjson%2Fcats.json%22%2C%20function(json)%20%7B%0A%0A%20%20%20%20%20%20%20%20var%20html%20%3D%20%22%22%3B%0A%20%20%20%20%20%20%20%20%2F%2F%20Only%20change%20code%20below%20this%20line.%0A%20%20%20%20%20%20%20%20json.forEach(function(val)%20%7B%0A%20%20%20%20%20%20%20%20%20%20var%20keys%20%3D%20Object.keys(val)%3B%0A%20%20%20%20%20%20%20%20%20%20html%20%2B%3D%20%22%3Cdiv%20class%3D'cat'%3E%22%3B%0A%20%20%20%20%20%20%20%20%20%20keys.forEach(function(key)%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20html%20%2B%3D%20%22%3Cstrong%3E%22%20%2B%20key%20%2B%20%22%3C%2Fstrong%3E%3A%20%22%20%2B%20val(key)%20%2B%20%22%3Cbr%20%2F%3E%22%3B%0A%20%20%20%20%20%20%20%20%20%20%7D)%3B%0A%20%20%20%20%20%20%20%20%20%20html%20%2B%3D%20%22%3C%2Fdiv%3E%3Cbr%20%2F%3E%22%0A%20%20%20%20%20%20%20%20%7D)%3B%0A%20%20%20%20%20%20%20%20%0A%20%20%20%20%20%20%20%20%0A%20%20%20%20%20%20%20%20%2F%2F%20Only%20change%20code%20above%20this%20line.%0A%0A%E2%80%89%E2%80%89%E2%80%89%E2%80%89%20%20%20%20%24(%22.message%22).html(html)%3B%0A%0A%E2%80%89%E2%80%89%20%20%20%20%7D)%3B%0A%20%20%20%20%7D)%3B%0A%20%20%7D)%3B%0Afcces%0A%0A%3Cdiv%20class%3D%22container-fluid%22%3E%0A%20%20%3Cdiv%20class%20%3D%20%22row%20text-center%22%3E%0A%20%20%20%20%3Ch2%3ECat%20Photo%20Finder%3C%2Fh2%3E%0A%20%20%3C%2Fdiv%3E%0A%20%20%3Cdiv%20class%20%3D%20%22row%20text-center%22%3E%0A%20%20%20%20%3Cdiv%20class%20%3D%20%22col-xs-12%20well%20message%22%3E%0A%20%20%20%20%20%20The%20message%20will%20go%20here%0A%20%20%20%3C%2Fdiv%3E%0A%20%20%3C%2Fdiv%3E%0A%20%20%3Cdiv%20class%20%3D%20%22row%20text-center%22%3E%0A%20%20%20%20%3Cdiv%20class%20%3D%20%22col-xs-12%22%3E%0A%20%20%20%20%20%20%3Cbutton%20id%20%3D%20%22getMessage%22%20class%20%3D%20%22btn%20btn-primary%22%3E%0A%20%20%20%20%20%20%20%20Get%20Message%0A%20%20%20%20%20%20%3C%2Fbutton%3E%0A%20%20%20%20%3C%2Fdiv%3E%0A%20%20%3C%2Fdiv%3E%0A%3C%2Fdiv%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/601.6.17 (KHTML, like Gecko) Version/9.1.1 Safari/601.6.17</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```html

<script>
  $(document).ready(function() {

    $("#getMessage").on("click", function() {
      $.getJSON("/json/cats.json", function(json) {

        var html = "";
        // Only change code below this line.
        json.forEach(function(val) {
          var keys = Object.keys(val);
          html += "<div class='cat'>";
          keys.forEach(function(key) {
            html += "<strong>" + key + "</strong>: " + val(key) + "<br />";
          });
          html += "</div><br />"
        });
        
        
        // Only change code above this line.

        $(".message").html(html);

      });
    });
  });
</script>

<div class="container-fluid">
  <div class = "row text-center">
    <h2>Cat Photo Finder</h2>
  </div>
  <div class = "row text-center">
    <div class = "col-xs-12 well message">
      The message will go here
   </div>
  </div>
  <div class = "row text-center">
    <div class = "col-xs-12">
      <button id = "getMessage" class = "btn btn-primary">
        Get Message
      </button>
    </div>
  </div>
</div>

```

### PR DESCRIPTION
**Pull request by dhcodes**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of FreeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) them into one commit).
- [X] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [X] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [X] Tested changes locally.
- [X] Closes currently open issue (replace XXXX with an issue no): Closes #9586

### CODE REVIEW

**Comment by atjonathan** [comment]
### **LGTM :+1:**

**Comment by Bouncey** [comment]
@dhcodes Can you please reapet the change for the different languages also.

**Comment by dhcodes** [comment]
Yeah, I'll do that @Bouncey

**Comment by dhcodes** [comment]
Okay, should be good now.

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,7 @@ pids
 logs
 results
 tmp
+.ds_store
```
Comment (atjonathan) [inline .gitignore]: This should be removed.

**Comment by atjonathan** [comment]
@dhcodes, please [squash](https://www.freecodecamp.com/wiki/en/git-squash/) your commits.

**Comment by dhcodes** [comment]
@atjonathan You mean going forward, right? Because I looked at rebase and there have been a lot of commits since mine so I don't think I can squash now. Sorry for my newbness.

**Comment by atjonathan** [comment]
@dhcodes just dropping those commits should be fine.

**Comment by dhcodes** [comment]
I think this is ready now...

**Comment by atjonathan** [comment]
@dhcodes you still have some unwanted commits. Try dropping those commits.

**Comment by atjonathan** [comment]
@dhcodes, I will squash these commits in-browser, thanks :smiley:

**Pull request merged**